README.md

```diff
@@ -0,0 +1,28 @@
+# Sushi LP Token Vested Reward Calculation
+
+## Setup
+```
+pip3 install -r requirements.txt
+```
+
+## Run
+execute the run.py program with parameter ```claim-date``` in the format of YYYY-MM-DD
+
+```
+# example
+python3 run.py --claim-date 2021-09-05
+```
```

Comment:
Please use the same argument as the https://github.com/sushiswap/sushi-vesting-query

> ## Options
> `-s` - Start Block, defaults to 10959148 (vesting start), defines the start of the period of the calculation
> 
> `-e` - End Block, required, defines the end of the period of the calculation
> 
> `-c` - Claim Block, default to current block height, defines the block of the snapshot of claimed amounts
>